FHEM/00_SIGNALduino.pm

```diff
@@ -2941,7 +2942,11 @@ sub SIGNALduino_IdList($@)
 	{
 		if ($wflag == 1)				# whitelist active
 		{
-			next if (!exists($WhitelistIDs{$id}))		# Id wurde in der whitelist nicht gefunden
+			if (!exists($WhitelistIDs{$id}))		# Id wurde in der whitelist nicht gefunden
+			{
+				push (@skippedWhiteId, $id);
+				next;
+			}
```

Comment:
wenn der user mit z.B. 3 in der whitelist nur die ID 3 aktivieren will, bekommt er nun eine log Ausgabe in der die restlichen ca 90 Protokolle aufgeführt sind. 
Dies ist meiner Meinung nach unnötig.

Anders sieht es aus, wenn er mit nicht aktiver whitelist alle Protokolle aktivieren will, dann bekommt er in den logs die Info welche Protokolle nicht aktiviert wurden.

Reply:
Dann lass es uns auf verbose 5 stellen